src/Microsoft.Security.Utilities.Core/CommonAnnotatedKey.cs

```diff
@@ -59,33 +59,36 @@ public static bool TryCreate(string key, out CommonAnnotatedKey secret)
             return true;
         }
 
-        private byte[] bytes;
-        private string base64Key;
+        public byte[] KeyBytes { get; protected set; }
```

Comment:
>KeyBytes [](http://example.com/codeflow?start=22&length=8)

The setter for either of these properties should update the other form as well, arguably.

So if you set the bytes, rewrite the encoded key and vice versa. We should probably do that if we expect classes to extend this case class.

Reply:
updated the setter logic to rewrite another property.